merkletree.go

```diff
@@ -123,7 +130,7 @@ insertLoop:
 				return
```

Comment:
fixed in https://github.com/coniks-sys/libmerkleprefixtree-go/blob/30123ed70d6ca2c67ff07e005a939fe92357f225/merkletree.go#L133